src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1322,6 +1324,42 @@ public static function getServiceConditionals($value)
     }
 
     /**
+     * Checks whether the requested file or directory exists and registers the result for resource tracking.
+     *
+     * @param string      $path          The file or directory path for which to check the existence
+     * @param bool|string $trackContents Whether to track contents of the given resource. If a string is passed,
+     *                                   it will be used as pattern for tracking contents of the requested directory
+     *
+     * @return bool
+     *
+     * @final
+     */
+    public function fileExists($path, $trackContents = true)
```

Comment:
@jakzal This follows up https://github.com/symfony/symfony/pull/21452#discussion_r98355205 which gives the arguments against the need for reflecting the tracking effect in the method name.